test/test_methods.py

```diff
@@ -149,6 +149,18 @@ def test_load_coordinates(self):
                 for coords, expected in zip(coordinates, objects):
                     self.assertEqual(coords, expected)
 
+    def test_load_coordinates_sci_notation(self):
```

Comment:
s/sci/scientific

Reply:
Sorry, I didn't understand :-) Do you mean rename the function by removing 'sci'?
Maybe calling it `test_load_coordinates`?

Reply:
Apologies for not being clear: I meant that we should call it 
`test_load_coordinates_scientific_notation`.

(`s/.../...` [comes from sed](https://www.gnu.org/software/sed/manual/html_node/The-_0022s_0022-Command.html)).

Reply:
Ah ok yes, I renamed it :-)